init.c

```diff
@@ -55,14 +55,18 @@
 #include <sys/wait.h>
 #include <sys/time.h>
 
-#define CHECK_PAGER \
-  if ((CurrentMenu == MENU_PAGER) && (idx >= 0) &&	\
-	    (MuttVars[idx].flags & R_RESORT)) \
-	{ \
-	  snprintf (err->data, err->dsize, \
-	    _("Not available in this menu.")); \
-	  return (-1); \
-	}
+static int check_pager(int idx, BUFFER *err)
+{
+  if ((CurrentMenu == MENU_PAGER) && (idx >= 0) && (MuttVars[idx].flags & R_RESORT))
+  {
+    snprintf(err->data, err->dsize, _("Not available in this menu."));
+    return (-1);
+  }
+  else
+  {
+    return 0;
+  }
+}
 
```

Comment:
Otherwise I'd change the semantics a bit of that function, to have it use a boolean:

```
static bool check_pager(int idx, BUFFER *err)
{
    if (A)
    {
        return false;
    }
    else
    {
        return true;
    }
```

And then do in the usage places below:

```
if (!check_pager(idx, err))
    return -1;
```